scripts/deploy.sh

```diff
@@ -363,6 +364,92 @@ EOF
   fi
   PAYG_DEPLOYMENT_2_BASE_URL="https://${payg2_fqdn}"
 
+  ptu_deployment_1_api_key=$SIMULATOR_API_KEY
+  payg_deployment_1_api_key=$SIMULATOR_API_KEY
+  payg_deployment_2_api_key=$SIMULATOR_API_KEY
+
+else
+
```

Comment:
As the script has got bigger, I wonder whether it's worth a comment here along the lines of "not using the simulator"?